project/CelebornBuild.scala

```diff
@@ -870,6 +885,8 @@ trait FlinkClientProjects {
   
         (assembly / assemblyMergeStrategy) := {
           case m if m.toLowerCase(Locale.ROOT).endsWith("manifest.mf") => MergeStrategy.discard
+          case m if m.startsWith("META-INF/license/") => MergeStrategy.discard
```

Comment:
why do we need to discard the files in folder `META-INF/license/`?

Reply:
The MR module depends on the netty 3x version, so there is a `META-INF/license/` directory
Check that Spark and Flink do not depend on netty 3x. This configuration is now removed.

https://repo1.maven.org/maven2/io/netty/netty/3.10.6.Final/netty-3.10.6.Final.jar

<img width="672" alt="image" src="https://github.com/apache/incubator-celeborn/assets/3898450/8be6e197-7343-416a-8cf9-a7dfc1f70b8b">